dao.py

```diff
@@ -166,6 +167,9 @@ def delete_player(self, player):
     def update_player(self, player):
         return self.players_col.update({'_id': player.id}, player.dump(context='db'))
 
+    def update_region(self, region):
+        return self.regions_col.update({'_id': region.id}, region.get_json_dict())
```

Comment:
it's been running and passing the tests, but I'll try this method for consistency